src/Schema/migrations/test.js

```diff
@@ -181,6 +181,10 @@ describe('migration step functions', () => {
       'type',
     )
   })
+  it('throws if destroyColumn() is malformed', () => {
```

Comment:
pls update one of the tests above to test for the format of what destroyColumn returns